src/features/home/pages/tests/Home.test.tsx

```diff
@@ -88,6 +93,7 @@ describe('Home component', () => {
       partialUser: { depositExpirationDate: new Date('2020-02-16T17:16:04.735235') },
     })
     expect(queryByText('Tu as 496 € sur ton pass')).toBeFalsy()
+    await act(flushAllPromises)
     expect(queryByText('Ton crédit est expiré')).toBeTruthy()
```

Comment:
Pourquoi tu n'as pas fait cela : 
```suggestion
    waitForExpect(() => {
      expect(queryByText('Tu as 496 € sur ton pass')).toBeFalsy()
      expect(queryByText('Ton crédit est expiré')).toBeTruthy()
    })
```

Reply:
C'est le 1er que j'ai test et le `flushAllPromises` à fonctionné ce qui n'était pas le cas pour les autres tests. Je pense qu'on peux laisser car le 1er est plus directe si il fonctionne.

Reply:
Si il n'y pas de soucis lié à act, je préfère le waitForExpect pour deux raisons : 
- On est consistent sur l'ensemble du fichier
- waitForExpect est plus pro que notre mini-hack custom :)